purchases/src/main/kotlin/com/revenuecat/purchases/PurchaserInfo.kt

```diff
@@ -166,6 +195,6 @@ data class PurchaserInfo internal constructor(
     */
     companion object {
 
-        internal const val SCHEMA_VERSION = 2
+        internal const val SCHEMA_VERSION = 3
```

Comment:
Increased this but I actually don't think we need to right? This will make sure if the cached PurchaserInfo is version 2, it will be discarded so we don't crash if we try to access something that's not in the JSON when building a PurchaserInfo from the cached JSON. Since the JSON hasn't really changed, I think we don't have to increase this, but I want to make sure I am not missing any case.

Reply:
Exactly, I prefer increasing every time we touch something, just in case we are missing something and we actually crash on startup when accessing the cache.